<launch>
  <node pkg="mpc_offboard"
        type="traj_publisher_mavros.py"
        name="trajectory_publisher_mavros"
        output="screen">
    <param name="waypoint_mode" value="square"/>
    <param name="square_center_n" value="0.0"/>
    <param name="square_center_e" value="0.0"/>
    <param name="square_size" value="20.0"/>
    <param name="square_altitude" value="-5.0"/>
    <param name="square_points_per_side" value="10"/>
    <param name="square_constant_yaw" value="false"/>

    <param name="ref_speed" value="2.0"/>
    <param name="loop_trajectory" value="true"/>
</node>
</launch>